packages/databricks-vscode/src/configuration/LoginWizard.ts

```diff
@@ -45,6 +46,13 @@ interface State {
 export class LoginWizard {
     private state = {} as Partial<State>;
     private readonly title = "Configure Databricks Workspace";
+    private _profiles: Array<ConfigEntry> = [];
+    async getProfiles() {
+        if (this._profiles.length === 0) {
```

Comment:
The idea is to only have static "run" method as the only public interface here, so this cache is only active during the run. This makes sense because we do not want to keep reloading the profiles in the middle of the login dialogue flow. 

> Does listing profiles visibly slows down the wizard process? We might as well just not have the cache
Yes, I am seeing latencies as high as 5s.